src/main/java/com/okta/sdk/clients/AuthApiClient.java

```diff
@@ -114,6 +115,19 @@ public AuthResult authenticate(String username, String password, String relaySta
     }
 
     /**
+     * Authenticate into organization.
+     *
+     * @param  token {@link String}                 activationToken to use for authentication.
+     * @return {@link AuthResult}                   Result of the authentication transaction.
+     * @throws IOException                          If an input or output exception occurred.
+     */
+    public AuthResult authenticate(String token) throws IOException {
+        Map<String, Object> params = new HashMap<String, Object>();
```

Comment:
This will require to change the sdk version of the project. (now is 1.5) This should be in different change.